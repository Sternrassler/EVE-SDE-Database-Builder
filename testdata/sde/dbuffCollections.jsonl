{"buffID":1,"collectionID":100,"filterGroupID":null,"filterTypeID":null,"locationGroupID":null,"locationTypeID":null,"modifierType":"locationModifier","modifyingAttributeID":4,"modifyingTypeID":null,"operation":"postMul","value":1.1}
{"buffID":2,"collectionID":101,"filterGroupID":25,"filterTypeID":null,"locationGroupID":null,"locationTypeID":null,"modifierType":"itemModifier","modifyingAttributeID":37,"modifyingTypeID":null,"operation":"postPercent","value":0.05}
{"buffID":3,"collectionID":102,"filterGroupID":null,"filterTypeID":587,"locationGroupID":null,"locationTypeID":null,"modifierType":"locationRequiredSkillModifier","modifyingAttributeID":9,"modifyingTypeID":null,"operation":"preAssignment","value":1.0}
