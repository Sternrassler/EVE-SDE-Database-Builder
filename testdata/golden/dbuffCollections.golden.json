[
  {
    "collectionID": 100
  },
  {
    "collectionID": 101
  },
  {
    "collectionID": 102
  }
]